EightBall.java

```diff
@@ -0,0 +1,17 @@
+package com.fortify.sca.plugins.maven.samples;
+
+import java.io.FileReader;
+
```

Comment:
<p><b>Security Scanning</b> / Fortify SAST</p>
<h3>Low - J2EE Bad Practices: Leftover Debug Code </h3>
<p>The class EightBall contains debug code, which can create unintended entry points in a deployed web application.


</p>
<br>
<p><a href=http://localhost:2320/ssc/html/ssc/version/42/fix/5886/?engineType=SCA&issue=BB9F74FFA0FF75C9921D0093A0665BEB&filterSet=f2f88863-0d8f-4d22-b001-3e1cd3ec0f45 target="_blank" rel="noopener noreferrer">More detailed information</a></p>